src/components/ha-icon-prev.ts

```diff
@@ -5,13 +5,14 @@ import { HaIcon } from "./ha-icon";
 
 export class HaIconPrev extends HaIcon {
   public connectedCallback() {
-    this.icon =
-      window.getComputedStyle(this).direction === "ltr"
-        ? "hass:chevron-left"
-        : "hass:chevron-right";
-
-    // calling super after setting icon to have it consistently show the icon (otherwise not always shown)
     super.connectedCallback();
+
+    setTimeout(() => {
+      this.icon =
+        window.getComputedStyle(this).direction === "ltr"
+          ? "hass:chevron-left"
+          : "hass:chevron-right";
+    }, 100);
```

Comment:
Please add a comment to each setTimeout.

Reply:
did you try moving this to `firstUpdated` instead of connectedCallback ?

Reply:
Currently HA-ICON is an HtmlElement

Reply:
`ha-icon` extends `iron-icon`, which is a Polymer element, so you should be able to override `ready()`